# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../spqmail/0005-qmail-sasl-depends-on-fork.h.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From b90a2eae7959fd189d02c966f9eca87f860ef038 Mon Sep 17 00:00:00 2001
From: Christian Wiese <christian.wiese@securepoint.de>
Date: Tue, 4 Dec 2012 10:19:43 +0100
Subject: [PATCH] Makefile: fixed parallel build by explicitely set that
 qmail-sasl.c depends on the generated fork.h

---
 Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ca54fad..537a0d7 100644
--- a/Makefile
+++ b/Makefile
@@ -2162,7 +2162,7 @@ compile out_log.c out_log.h
 	./compile out_log.c
 
 qmail-sasl.o: \
-compile qmail-sasl.c qmail-sasl.h
+compile qmail-sasl.c qmail-sasl.h fork.h
 	./compile qmail-sasl.c
 
 realrcptto.o: \
-- 
1.7.10.2

